jwst/firstframe/firstframe_step.py

```diff
@@ -0,0 +1,29 @@
+from ..stpipe import Step
+from .. import datamodels
+from . import firstframe_sub
+
+
+class FirstFrameStep(Step):
+    """
+    FirstFrameStep: This is a MIRI specific task.  If the number of groups
+    is greater than 1, the GROUP data quality flags for the first group will
+    be set to DO_NOT_USE.
+    """
+
+    def process(self, input):
+
+        # Open the input data model
+        with datamodels.open(input) as input_model:
+
+            # check the data is MIRI data
+            detector = input_model.meta.instrument.detector
```

Comment:
Couldn't hurt to make sure we're dealing with uppercase for the purpose of the comparison in line 20, so add the "upper" method here:
```
 detector = input_model.meta.instrument.detector.upper()
```

Reply:
Done.